shim/shim.py

```diff
@@ -855,14 +856,19 @@ def main() -> None:
                 binary, arguments, apk_directory, dex_directory
```

Comment:
damn it, this also needs to be changed, but it's facebook-internal code. I will need to change it myself before merging the PR.